.ci/jobs.yml

```diff
@@ -1,21 +1,34 @@
 JOB:
-  - intake
-  - firefoxSmoke
+  - kibana-intake
+  - x-pack-intake
+  - kibana-firefoxSmoke
   - kibana-ciGroup1
   - kibana-ciGroup2
   - kibana-ciGroup3
   - kibana-ciGroup4
   - kibana-ciGroup5
   - kibana-ciGroup6
-  # - kibana-visualRegression
+  - kibana-ciGroup7
+  - kibana-ciGroup8
+  - kibana-ciGroup9
+  - kibana-ciGroup10
+  - kibana-ciGroup11
+  - kibana-ciGroup12
+  - kibana-visualRegression
```

Comment:
Let's leave the visualRegression tests disabled for now. @liza-mae is working to re-enable them